test/e2e/main.go

```diff
@@ -67,11 +67,12 @@ func runSuite() error {
 	}
 	glog.Info("PASS: CreateClusterAutoscaler")
 
-	if err := ExpectClusterAutoscalerAvailable(); err != nil {
-		glog.Errorf("FAIL: ExpectClusterAutoscalerAvailable: %v", err)
-		return err
-	}
-	glog.Info("PASS: ExpectClusterAutoscalerAvailable")
+	// TODO: Disabled temporarily
```

Comment:
can you add some context on why to the TODO?

Reply:
Should be good now.